aten/src/ATen/native/quantized/cpu/qnnpack/src/q8conv/8x8-aarch64-neon.S

```diff
@@ -31,17 +50,27 @@ BEGIN_FUNCTION pytorch_q8conv_ukernel_8x8__aarch64_neon
 
     # Load bias0123, bias4567
     LD1 {v8.4s, v9.4s}, [x5], 32
-    MOV x9, 2
+
+    # Load output channel index
+    # Note since this is an offset into a byte pointer
+    # We do not need to multiply with size of pointer type
+    LDR x9, [sp]
+    # Load pointer to per channel zero points array
+    LDR x10, [x8]
```

Comment:
How about replacing this with `LDR x10, [x8], 8` and getting rid of `ADD x8, x8, 8` below.

Reply:
Yeah tried that. Didn't compile. It is a valid instruction but does not compile on android.

Reply:
That's strange.  I did a quick grep on XNNPACK to see if they have any such usage and there is:

https://github.com/google/XNNPACK/blob/8bd4cbae92d8080272fcae3aa0509ad49375bbe0/src/f32-gemm/4x12-aarch64-neonfma-cortex-a53.S.in#L176

It's not a big deal.  Will barely make a difference in isolation but I was hoping we could avoid it.  At least this is not in the inner most loop - although it is in a loop - the one that goes over the tiles.